tests/network/localnet/liblocalnet.py

```
@@ -42,6 +47,20 @@ def create_traffic_client(
     )
 
 
+def create_jumbo_frame_traffic_client(
+server_vm: BaseVirtualMachine, client_vm: BaseVirtualMachine, spec_logical_network: str, maximum_segment_size: int
```

Flake8 (7.2.0)

[error] continuation line missing indentation or outdented

(E122)